### DISCUSSION TITLE
Get active / actual url in livewire:navigated event

### DISCUSSION THREAD

**Discussion by codevApp** [livewire/livewire#8012] [1. Help] [answered]
Hi,

is it possible to retrieve the current url in the l**ivewire:navigated** event or after executing **wire:navigate**?
Only the previous one is displayed in window.location.href.

`    document.addEventListener('livewire:navigated', () => {
        console.log(window.location.href)
    })`

The current page must somehow be kept in the Navigate event?

**Comment by PhiloNL** [comment, marked as answer]
It's a bit of a hack but this works: https://wirebox.app/b/4jd01

```
document.addEventListener('livewire:navigated', () => { 
            setTimeout(() => {
                console.log(window.location.href) 
            }, 1);
        })
```

It seems `livewire:navigated` is triggered to early: https://github.com/livewire/livewire/blob/863aabab71d84bfa3b742fc265b11de57cd0ff3d/js/plugins/navigate/index.js#L81-L83

Added a PR: https://github.com/livewire/livewire/pull/8015

**Edit**: PR got merged so this should be fixed in the next release :)
